parlai/tasks/light_dialog/worlds.py

```diff
@@ -91,6 +91,7 @@ def parley(self):
             act['id'] = 'persona'
             agents[0].observe(validate(act))
         act = deepcopy(agents[0].act())
+        acts[0] = act
```

Comment:
is this ever used anywehre?

Reply:
yes, it's used in the display function of the world

Reply:
didn't realize, thanks